operations/helm/scripts/build.sh

```diff
@@ -59,9 +62,18 @@ for FILEPATH in $TESTS; do
   INTERMEDIATE_OUTPUT_DIR="${INTERMEDIATE_PATH}/${TEST_NAME}-generated"
   OUTPUT_DIR="${OUTPUT_PATH}/${TEST_NAME}-generated"
 
+  echo ""
   echo "Templating $TEST_NAME"
+  ARGS=("${TEST_NAME}" "${CHART_PATH}" "-f" "${FILEPATH}" "--output-dir" "${INTERMEDIATE_OUTPUT_DIR}" "--namespace" "citestns")
+
+  echo -n "Checking for kubeVersionOverride..."
```

Comment:
```suggestion
  echo "Checking for kubeVersionOverride..."
```
I think better to keep the newline.  Otherwise the following line will be appended to the same line make it look like below.
```
Checking for kubeVersionOverride...Warning: injecting Kubernetes version override: kubeVersionOverride=1.20
```